lora_diffusion/preprocess_files.py

```diff
@@ -278,8 +285,10 @@ def load_and_save_masks_and_captions(
     images = [Image.open(file) for file in files]
 
     # captions
-    print(f"Generating {len(images)} captions...")
-    captions = blip_captioning_dataset(images, text=caption_text)
+    captions = caption_text
+    if not isinstance(caption_text, List):
```

Comment:
`caption_text` is used three times here but was changed to `captions_text` in the function. Changing these ooccurrences to `captions_text` will resolve the problem.